config/routes.rb

```diff
@@ -1,6 +1,10 @@
-Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+# frozen_string_literal: true
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+Rails.application.routes.draw do
+  root 'users#index'
+  get 'users/index', to: 'users#index'
+  get 'users/:user_id', to: 'users#show'
+  get 'users/:user_id/posts', to: 'posts#index'
+  get 'users/:user_id/posts/:id', to: 'posts#show'
```

Comment:
- [ ] Good job so far. I'm so impressed with how good you're on your first rails project 🔥. Let's just fix one little thing here okay? let's remove the `index` in the users endpoint, and let's simply have `users` so that once a user goes to `localhost:3000` or `localhost:3000/users`, our users index action renders and not `localhost:3000/users/index`

```suggestion
  get '/users', to: 'users#index'
  get '/users/:user_id', to: 'users#show'
  get '/users/:user_id/posts', to: 'posts#index'
  get '/users/:user_id/posts/:id', to: 'posts#show'
```

Reply:
Thank you @julie-ify